libs/sdk-backend-spi/src/organization/settings/index.ts

```diff
@@ -9,9 +9,16 @@ import { IWhiteLabeling } from "@gooddata/sdk-model";
  */
 export interface IOrganizationSettingsService {
     /**
-     * Set whiteLabeling in organization.
+     * Set whiteLabeling for organization.
      *
      * @returns promise
      */
     setWhiteLabeling(whiteLabeling: IWhiteLabeling): Promise<void>;
+
+    /**
+     * Set locale for organization.
+     *
+     * @returns promise
```

Comment:
Workspace and user has a better JSDoc with described parameter. Please, use it here as well.

Reply:
ok, done